sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -113,7 +115,7 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("cast string to timestamp") {
-    ALL_TIMEZONES.par.foreach { tz =>
+    new ParVector(ALL_TIMEZONES.toVector).foreach { tz =>
```

Comment:
`ALL_TIMEZONES` is `Seq` here. Maybe, `ParSeq` instead of `ParVector`?

Reply:
ParSeq is a trait unfortunately. I think this is the right impl for this case.